side_length: .0777875 #Unit is in meters, equivalent to 3.0625 inches
focal_length: 639
